tests/Unit/Domain/DomainTestHelpers.cpp

```diff
@@ -394,23 +452,47 @@ tnsr::i<DataType, SpatialDim> unit_basis_form(
 /// \cond
 #define DIM(data) BOOST_PP_TUPLE_ELEM(0, data)
 
-#define INSTANTIATE(_, data)                                                \
-  template boost::rational<size_t> fraction_of_block_volume(                \
-      const ElementId<DIM(data)>& element_id) noexcept;                     \
-  template void test_domain_construction<DIM(data)>(                        \
-      const Domain<DIM(data)>& domain,                                      \
-      const std::vector<DirectionMap<DIM(data), BlockNeighbor<DIM(data)>>>& \
-          expected_block_neighbors,                                         \
-      const std::vector<std::unordered_set<Direction<DIM(data)>>>&          \
-          expected_external_boundaries,                                     \
-      const std::vector<std::unique_ptr<domain::CoordinateMapBase<          \
-          Frame::Logical, Frame::Inertial, DIM(data)>>>&                    \
-          expected_maps) noexcept;                                          \
-  template void test_initial_domain(                                        \
-      const Domain<DIM(data)>& domain,                                      \
-      const std::vector<std::array<size_t, DIM(data)>>&                     \
-          initial_refinement_levels) noexcept;                              \
-  template void test_physical_separation(                                   \
+#define INSTANTIATE(_, data)                                                   \
```

Comment:
Are the only differences between these two instantiations the frame? if so, can you just add a parameter to the instantiation generation?